model/task_lifecycle_test.go

```diff
@@ -1191,47 +1196,84 @@ func TestUpdateBuildStatusForTask(t *testing.T) {
 				{Status: evergreen.TaskUndispatched, Activated: true},
 				{Status: evergreen.TaskUndispatched, Activated: true},
 			},
-			expectedBuildStatus:   evergreen.BuildCreated,
-			expectedVersionStatus: evergreen.VersionCreated,
-			expectedPatchStatus:   evergreen.PatchCreated,
+			expectedBuildStatus:       evergreen.BuildCreated,
+			expectedVersionStatus:     evergreen.VersionCreated,
+			expectedPatchStatus:       evergreen.PatchCreated,
+			expectedBuildActivation:   true,
+			expectedVersionActivation: true,
+			expectedPatchActivation:   true,
+		},
+		"created / deactivated": {
```

Comment:
I think adding a `/` to the test name might mess with the `gotest.parse_file` logic and treat "deactivated" as a subtest of "created" (or might just visually look confusing since it'll appear in the Go test output as `TestUpdateBuildStatusForTask/created_/_deactivated` due to spaces being auto-replaced).

Reply:
oh fair haha i'll change it to or